### PR TITLE
add discord links

### PR DESCRIPTION
**Pull request by KotlinIsland**
TODO: insert amongus referrence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,26 +26,25 @@ Basedmypy can be installed using pip from PyPI or from this GitHub repo:
     python -m pip install -U basedmypy
 
 ### Running
-Basedmypy currently overrides the `mypy` installation:
+Basedmypy is a direct alternative to the `mypy` installation:
```
Comment (DetachHead) [inline README.md:29]: this seems misleading. it does override the mypy installation, which i think is a limitation that's very important to mention

File: setup.py [hunk 2]
```diff
@@ -201,7 +201,6 @@ def run(self):
       description=description,
       long_description=long_description,
       author='KotlinIsland',
-      url='https://www.github.com/KotlinIsland/basedmypy',
```
Comment (DetachHead) [inline setup.py:204]: why was this removed

Comment (KotlinIsland) [reply setup.py:204]: image
It's linked as the "Homepage", which is more misleading imo.

File: README.md (hunk 1 above)
Comment (KotlinIsland) [reply README.md:29]: hmm

Comment (KotlinIsland) [reply README.md]: Hows that look?

Comment (DetachHead) [reply README.md]: that's ok i guess but i think it was fine how it was originally, less verbose and more clear imo

Comment (KotlinIsland) [reply README.md]: I feel like "overrides" could be misleading, indicating that in needs to override mypy to be installed or something.

Comment (KotlinIsland) [reply README.md]: "in place of" makes more sense

Comment (DetachHead) [reply README.md]: > I feel like "overrides" could be misleading, indicating that in needs to override mypy to be installed or something.

but it does, doesn't it? though i guess "in place of" means pretty much the same thing so either is fine ig
